Assets/MixedRealityToolkit.Tests/PlayModeTests/InteractableTests.cs

```diff
@@ -627,6 +629,58 @@ public IEnumerator TestPressableToggleHoloLens2()
             GameObject.Destroy(interactable.gameObject);
         }
 
+        [UnityTest]
+        /// <summary>
+        /// Test InteractableToggleCollection CurrentIndex updates
+        /// </summary>
+        public IEnumerator TestInteractableToggleCollectionIndexUpdate()
+        {
+            InteractableToggleCollection interactableToggleCollection;
+            int numRadials = 6;
+
+            AssembleInteractableToggleCollection(
+                out interactableToggleCollection,
+                numRadials,
+                Vector3.forward);
+
+            var toggleList = interactableToggleCollection.ToggleList;
+
+            int[] onClickEventCalled = new int[numRadials];
+
+            int index = 0;
+            foreach (var radial in toggleList)
```

Comment:
nit: I'd suggest simplifying this to for (int index = 0; blah blah), foreach purpose is defeated here